### ISSUE TITLE
Use Diátaxis Framework to restructure SSVC documentation

### ISSUE THREAD

**Issue by ahouseholder** [CERTCC/SSVC#294]
The SSVC documentation has grown, and it's becoming unwieldy to try to fit everything into a single document that suits everyone's needs. Also, our experience with the CVD Guide and other such guidance shows that large PDFs tend to limit the audience to only the most dedicated readers.

_This_ issue is to open the conversation about adopting the [Diátaxis Framework](https://diataxis.fr/) to guide further development of the SSVC documentation. I intend to open a separate issue (edit: #295) to also propose making a website based on these principles as the primary SSVC documentation vehicle going forward. 

We are currently using the Diátaxis Framework to build documentation for the Vultron CVD Protocol, and it has been quite helpful in providing a clear top-level information architecture for documentation aimed at different audiences.

In a nutshell, the idea is that there are four kinds of documentation organized in two dimensions. Excerpting from [the compass](https://diataxis.fr/compass/):

| If the content describes… | …and serves the user’s… | …then it must belong to… |
| -- | -- | -- |
| practical steps | study | a tutorial |
| practical steps | work | a how-to guide |
| theoretical knowledge | work | reference |
| theoretical knowledge | study | explanation |

At this point, we'd be interested in any comments on adopting this approach.

**Comment by j---** [comment]
Possible we can leverage some of the community content in #291 to help with tutorials and how-to guides. 

This is a great idea.

**Comment by jeroenh** [comment]
Agreed! It took me a while to get through the whole SSCV PDF, and this format is indeed not the most accessible for practitioners to get started on it. I am intrigued by the framework you mentioned, it seems like a great way to approach this, happy to help out!
